src/app/visual/image/image-visual.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="sky-image-visual">
+  <sky-image
+    imageSource="~/assets/demo-img.png"
+    imageAlt="image for visual test"
+    caption="default caption">
```

Comment:
Closing `>` should be on its own line.
```
  caption="default caption"
>
</sky-image>
```
See: https://docs.blackbaud.com/skyux-team-docs/style#component-html-tags